nemo_text_processing/inverse_text_normalization/he/data/numbers/digit.tsv

```diff
@@ -0,0 +1,20 @@
+אחד	1
+שניים	2
+שני	2
+שלושה	3
+ארבעה	4
+חמישה	5
+שישה	6
+שבעה	7
+שמונה	8
+תשעה	9
+אחת	1
+שתיים	2
```

Comment:
Yes. The digits doesn't change there are different ways to say it. In Hebrew, numbers usually have gender, so we wanted to catch both masc. numbers and fem. numbers. In addition, some numbers are said differently like 2, when coming in construct state (proximity). For example "two hands" would be  "שתי ידיים" and not " שתיים ידיים".
We wanted to include all the forms of the numbers, for better covering test cases.

Reply:
Ah gotcha, and the infixation would make this a pain to just include the gendering in the graph. Could I bother you to split the data list into something like `digit_masc` `digit_fem`? It'll be easier for a maintainer to return to and understand without knowledge of  the language.

Reply:
There is no easy way to include the gendering in the graph, because there is no basic rule to follow.
In addition, I think it would actually be more confusing and harder to maintain if we start splitting the digits to different lists, because they all used in the same places in the graphs. Maintaining different lists won't be easier in my opinion. I can do that, but I think that would not help maintainers in the future

Reply:
It'd be preferred from a non-speaker end. Because of the amount of language knowledge that goes into these things, we want to make sure there's as much surface level knowledge available. It's just easier on my end to pass off to another maintainer this way.